haiku-apps/bescreencapture/bescreencapture-2.2.1.recipe

```diff
@@ -15,9 +15,10 @@ COPYRIGHT="2014-2016 Stefano Ceccherini"
 LICENSE="BSD (3-clause)
 	MIT"
 REVISION="1"
-SOURCE_URI="https://github.com/jackburton79/bescreencapture/archive/v$portVersion.tar.gz"
-CHECKSUM_SHA256="779a7ac4e4e7b935f31ff95b47ab1325a81a98d9fe5d76017fb8c4e01b3ed5ac"
-SOURCE_FILENAME="bescreencapture-$portVersion.tar.gz"
+SOURCE_URI="https://github.com/jackburton79/bescreencapture/archive/v"$portVersion"a.tar.gz"
+CHECKSUM_SHA256="d36f34d0dcf8bb87d58d782e649f29862fbbaed68a26182ccb695b0b0716ec87"
+SOURCE_FILENAME="bescreencapture-"$portVersion"a.tar.gz"
+SOURCE_DIR="bescreencapture-"$portVersion"a"
```

Comment:
>  I'd like the version to remain 2.2.1, not 2.2.1a.

OK, no problem ;-)

BTW, would you mind making a small neutral change? I think this would be slightly better:
```
SOURCE_URI="https://github.com/jackburton79/bescreencapture/archive/v${portVersion}a.tar.gz"
+CHECKSUM_SHA256="d36f34d0dcf8bb87d58d782e649f29862fbbaed68a26182ccb695b0b0716ec87"
+SOURCE_FILENAME="bescreencapture-${portVersion}a.tar.gz"
+SOURCE_DIR="bescreencapture-${portVersion}a"
```

I know that using «**`"$portVersion"a`**» works fine here, but IMHO «**`${portVersion}a`**» is more consistent with the style of all other recipes that already use «**`${portVersion}`**». Thanks!